libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt

```diff
@@ -163,4 +165,36 @@ internal class MapboxDirectionsSession(
     override fun shutdown() {
         router.shutdown()
     }
+
+    /**
+     * Temporary method for handle routes with multiple waypoints.
+     * Current Directions API doesn't support more than one waypoint
+     * for [DirectionsCriteria.PROFILE_DRIVING_TRAFFIC].
+     * Also NavNative returns "null" in "routeOptions" field.
+     * After this two points will fixed - this method can be removed.
```

Comment:
```suggestion
     * After these two points are fixed - this method can be removed.
```

BTW can we add a `TODO` comment so we don't forget / is more discoverable?

Reply:
👍